app/views/checklist_tests/show.html.erb

```diff
@@ -1,112 +1,42 @@
-                        <% end %>
-                        <td><%= checklist_test_criteria_attribute(measure, criteria) %></td>
-                        <td><%= render partial: 'field_values', locals: { field_values: criteria['field_values'], negation: criteria['negation_code_list_id'], result: criteria['value']} if criteria['field_values'] || criteria['negation_code_list_id'] || criteria['value'] %></td>
-                        <% if coded_attribute?(criteria) %>
-                          <td>
-                            <% if criteria_field.object.complete?.nil? %>
-                              <i class = 'fa fa-fw fa-play-circle text-info invisible' aria-hidden="true"></i>
-                            <% elsif criteria_field.object.attribute_complete %>
-                              <i class = 'fa fa-fw fa-check text-success' aria-hidden="true"></i>
-                            <% else %>
-                              <i class = 'fa fa-fw fa-times text-danger' aria-hidden="true"></i>
-                            <% end %>
-                            <%= criteria_field.text_field :attribute_code, hide_label: true %>
-                          </td>
-                        <% elsif criteria['value'] && criteria['value'].type != 'CD' || criteria['field_values'] %>
-                          <td>
-                            <% if criteria_field.object.complete?.nil? %>
-                              <i class = 'fa fa-fw fa-play-circle text-info invisible' aria-hidden="true"></i>
-                            <% elsif criteria_field.object.result_complete %>
-                              <i class = 'fa fa-fw fa-check text-success' aria-hidden="true"></i>
-                            <% else %>
-                              <i class = 'fa fa-fw fa-times text-danger' aria-hidden="true"></i>
-                            <% end %>
-                            <%= criteria_field.text_field :recorded_result, hide_label: true %>
-                          </td>
-                        <% else %>
-                          </td>
-                        <% end %>
-                      </tr>
-                    <% end %>
-                  <% end %>
-                </tbody>
-              </table>
-            </div>
-            <div class = 'panel-footer'>
-              <%= f.submit 'Save', class: 'btn btn-default' %>
-              <span class = 'small-indent'>this saves the entire checklist page</span>
-            </div>
-          </div>
-        </div>
+  <div class = 'test-steps'>
+    <div class="panel panel-info">
+      <div class="panel-heading">
+        <h1 class="panel-title test-step">
+          <%= 1 %> <i class="fa fa-fw fa-bolt" aria-hidden="true"></i> <%= 'Checklist Instructions' %>
+        </h1>
       </div>
-    <% end %>
-  <% end %>
-<% end %>
-
+      <div class="panel-body">
+        <%= render partial: 'checklist_instructions', locals: { instructions: APP_CONFIG['tests']['ChecklistTest']['instructions'] } %>
```

Comment:
`checklist_instructions` also has a `.panel-body`, so this creates a `.panel-body` nested within a `.panel-body`.